src/main/java/uk/gov/hmcts/darts/arm/service/impl/ExternalObjectDirectoryServiceImpl.java

```diff
@@ -28,4 +30,13 @@ public List<ExternalObjectDirectoryEntity> findFailedStillRetriableArmEods(Pagea
         );
     }
 
+    @Override
+    public boolean hasNotAllMediaBeenCopiedFromInboundStorageForAtsProcessing(List<MediaEntity> mediaRequestFiles) {
```

Comment:
I initially called this method: `hasNotAllMediaBeenCopiedFromInboundStorage` but then I though this would be misleadiing. This method is not really reusable because its query aspects very specific to ats processing. 
But I can remove `ForAtsProcessing` if this generates confusion

Reply:
The service removes some complexity from the `AudioTransformationServiceImpl` both in production and test code